include/despf.inc.sh

```diff
@@ -158,6 +158,7 @@ getamx() {
         lookuphost=$(echo $ahost | cut -d\/ -f1)
       fi
     fi
+    mech=$(echo $mech | tr [A-Z] [a-z])
```

Comment:
Must use comma in tr, like tr '[A-Z]' '[a-z]'.
Without it it, there will be a shell expansion and may have undesired results.
```
echo A | tr [A-Z] [a-z]
a
izeca:tmp jcbf$ touch o
izeca:tmp jcbf$ echo A | tr [A-Z] [a-z]
o
izeca:tmp jcbf$ echo A | tr '[A-Z]' '[a-z]'
a
```